static/app/views/relocation/relocation.spec.tsx

```diff
@@ -0,0 +1,84 @@
+import {initializeOrg} from 'sentry-test/initializeOrg';
+import {render, screen, userEvent} from 'sentry-test/reactTestingLibrary';
+
+import Relocation from 'sentry/views/relocation/relocation';
+
+describe('Relocation', function () {
+  function renderPage(step) {
+    const routeParams = {
+      step,
+    };
+
+    const {routerProps, routerContext, organization} = initializeOrg({
+      router: {
+        params: routeParams,
+      },
+    });
+
+    render(<Relocation {...routerProps} />, {
+      context: routerContext,
+      organization,
+    });
+  }
+  describe('Get Started', function () {
+    it('renders', async function () {
+      renderPage('get-started');
+      expect(
+        await screen.findByText('Basic Information Needed to Get Started')
+      ).toBeInTheDocument();
+      expect(
+        await screen.findByText(
+          'In order to best facilitate the process some basic information will be required to ensure sucess with the relocation process of you self-hosted instance'
```

Comment:
My 2 cents: matching on specific paragraph text is a bit too much. Just match on the header and the forum fields that the user will actually need to interact with, plus a verification that SOME KIND of intro paragraph exists. But the specific text is a bit much.

Reply:
Looking at the other test cases, guess you're right. I'm used to in the past writing snapshot tests so that's where this style comes from